examples/pxScene2d/src/jsbindings/rcvrcore/rpcContext.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Created by tjcarroll2
+ * on 2/6/16.
+ */
+
+function RPCContext(theScene) {
+  var scene = theScene;
+  var rpcController = null;
+  var sourceTargetName;
+  var appNameTargetRegistered = false;
+  var functions = {};
+  var onRPCCall;
+
+  this.registerApp = function(targetName) {
```

Comment:
Bob called it appId on the Confluence page.  I think I like appNamespace better, since it implies it purpose.  I was originally looking at a general purpose RPC target that was more object based, but since we're taking the general onRPCCall approach this doesn't make sense anymore.